packages/ses-ava/src/types.js

```diff
@@ -0,0 +1,64 @@
+// @ts-check
+import 'ses';
+
+/**
+ * @callback Logger
+ * @param {...any} args
+ * @returns {void}
+ */
+
+/**
+ * @callback LogCallError
```

Comment:
any particular reason to document this type so far away from the implementation? is it re-used somewhere?